README.md

```diff
@@ -26,3 +26,23 @@ For more information, see the examples in `smt-switch/tests/btor`. Other useful
 * `smt-switch/include/solver.h`: this is the main interface you will be using
 * `smt-switch/include/ops.h`: this contains all the ops you might need
   * Note: create indexed ops like `Op(Extract, 7, 4)`
+
+## Python bindings
+To build the `cosa2` python bindings, first make sure that you have [Cython](https://cython.org/) version >= 0.29 installed. Then ensure that `smt-switch` and its python bindings are installed. Finally, you can configure with `./configure.sh --python` and then build normally. The sequence of commands would be as follows:
+```
+# Optional recommended step: start a python virtualenv
+# If you install in the virtualenv, you will need to activate it each time before using pycosa2
+# and deactivate the virtualenv with: deactivate
+python3 -m venv env
+source ./env/bin/activate
+pip install Cython==0.29 pytest
+./contrib/setup-smt-switch.sh --python
+pip install -e ./deps/smt-switch/build/python
```

Comment:
A minor glitch in the steps:
`./contrib/setup-btor2tools.sh`
must be run before
`./configure.sh --python`

Reply:
Good catch, thanks! Updated with: d7350f544ac067ba956aee1e745fe8806ba7c9a2